lua/telescope/actions/init.lua

```diff
@@ -619,8 +619,8 @@ local entry_to_qf = function(entry)
   return {
     bufnr = entry.bufnr,
     filename = from_entry.path(entry, false),
-    lnum = entry.lnum,
-    col = entry.col,
+    lnum = utils.get_default(entry.lnum, 1),
```

Comment:
I've was reading quite a lot of `builtin/`, `make_entry.lua` today (for readme, missing opts) and saw that we still have quite a lot of usage from these old `utils`. We should move towards `vim.F.if_nil`. And maybe plan other refactors. Clean up old code.

I might do this after the readme rewrite

Reply:
Yeah, I've stuck with `utils.get_default` to remain consistent with rest of telescope.  I'll quickly change this to `vim.F.if_nil` then.

Reply:
Yeah i wanna move towards utils if they are available in vim and remove them here.

I know that you know about `vim.F.if_nil`, i saw you use it, i just wanted to mention it :laughing: